source/index.html

```diff
@@ -1,12 +1,158 @@
+    </header>
 
+    <main class="main-container">
+      <section class="hero">
+        <h1 class="hero_title main-title">Функциональное питание для котов</h1>
+        <span class="hero__slogan">Занялся собой? Займись котом!</span>
+        <img class="hero__image" src="img/can-hero.png" width="280" height="270" alt="Cat Energy PRO 500 г Курица">
+        <a href="form.html" class="primary-button hero__button" href="form.html">Подобрать программу</a>
+      </section>
+
+      <section class="program">
+        <h2 class="visually-hidden">Наши программы</h2>
+
+        <section class="program__item">
+          <h3 class="program__title program__title--slim">Похудение</h3>
+          <p class="program__descr">
+            Ваш кот весит больше собаки и почти утратил способность лазить
+            по деревьям? Пора на диету! Cat Energy Slim поможет вашему
+            питомцу сбросить лишний вес.
+          </p>
+          <a class="program__link" href="#">Каталог Slim</a>
+        </section>
+
+        <section class="program__item">
+          <h3 class="program__title program__title--pro">Набор массы</h3>
+          <p class="program__descr">
+            Заработать авторитет среди дворовых котов и даже собак?
+            Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!
+          </p>
+          <a class="program__link" href="#">Каталог Pro</a>
+        </section>
+      </section>
+
+      <section class="about">
+        <h2 class="secondary-title about__title">Как это работает</h2>
+        <ul class="about__list">
+          <li class="about__item">
+              Функциональное питание содержит только полезные питательные вещества.
+          </li>
+
+          <li class="about__item">
+              Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
+          </li>
+
+          <li class="about__item">
+              Замените один-два приема обычной еды на наше функциональное питание.
+          </li>
+
+          <li class="about__item">
+              Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+          </li>
+        </ul>
+      </section>
+
+      <section class="example">
+        <h2 class="secondary-title example__title">Живой пример</h2>
+        <p class="example__result-descr">
+          Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на
+          Cat Energy Slim. Отличный результат без изнуряющих тренировок!
+          При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+        </p>
+
+        <div class="example__result-data-wrapper">
+          <dl class="result-data">
+            <dt class="result-data__parameter">5 кг</dt>
+            <dd class="result-data__descr">Снижение веса</dd>
+          </dl>
+
+          <dl class="result-data">
+            <dt class="result-data__parameter">60 дней</dt>
+            <dd class="result-data__descr">Затрачено времени</dd>
+          </dl>
+        </div>
```

Comment:
Можно же один список сделать ))
Фокус в том, что в отличие от `ul` и `ol` в этом виде списка можно пары дополнительно оборачивать дивами.

```suggestion
        <dl class="example__result-data-wrapper">
          <div class="result-data">
            <dt class="result-data__parameter">5 кг</dt>
            <dd class="result-data__descr">Снижение веса</dd>
          </div>

          <div class="result-data">
            <dt class="result-data__parameter">60 дней</dt>
            <dd class="result-data__descr">Затрачено времени</dd>
          </div>
        </dl>
```

Reply:
Я конечно может зря сразу не сказал обратить внимание, но `dt` — это то, что мы определяем, а `dd` — это значение.
Тут скорее должно быть наоборот, например "Снижение веса" это `dt`, а "5 кг" уже `dd`.

Здесь кроется и второй подвох. В разметке обязательно сперва должен идти `<dt>`.
Итого получается, что нам нужно в разметке поставить их в обратном порядке. А уже потом стилями расположить как нам надо визуально.